### DISCUSSION TITLE
file source, do not decompress

### DISCUSSION THREAD

**Discussion by luisloros** [vectordotdev/vector#17620] [Q&A] [answered]
Hi Vector team,

We have an use case where we want to push compressed files directly to S3.
We are seeing that Vector is opening those compressed files (as described [here](https://vector.dev/docs/reference/configuration/sources/file/#compressed-files)). In our case, we don't need to parse the logs inside those files, we just want to store them for eventual offline analysis.

Would it be possible for Vector not to open those files and just push them to S3 directly?

**Comment by jszwedko** [comment, marked as answer]
Hi @luisloros ! If you just want to copy files to S3, I think the AWS CLI would be a better bet. You can use `aws s3 sync` or `aws s3 cp` to copy files to S3.
